partner_invoice_target/models/account_invoice.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models,fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    # override vat to add translation
+    vat = fields.Char()
```

Comment:
What is this for?

Reply:
Hi @AungKoKoLin1997 , the vat field that i ask to be override is the one on res.partner model